pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUmbrellaStudyCached.java

```diff
@@ -190,7 +190,7 @@ public long insert(String studyName, String studyGuid, long umbrellaId, String w
                        Long precisionId, boolean shareLocation, String studyEmail, String recaptchaSiteKey) {
         long studyId = delegate.insert(studyName, studyGuid, umbrellaId, webBaseUrl, auth0TenantId, irbPassword, precisionId, shareLocation,
                 studyEmail, recaptchaSiteKey);
-        notifyModelUpdated(ModelChangeType.UMBRELLA, umbrellaId);
+        notifyModelUpdated(ModelChangeType.STUDY, studyId);
```

Comment:
If we're inserting a new study, is the "model" that changed a STUDY or UMBRELLA? Again, not familiar with this.